internal/test/framework/windowsvm.go

```diff
@@ -153,14 +153,11 @@ func (f *TestFramework) newWindowsVM(vmCount int) ([]TestWindowsVM, error) {
 		}
 		creds := types.NewCredentials(instanceID, ipAddress, types.Username)
 		winVM.Credentials = creds
-		log.Print("setting up ssh and winrm")
+		log.Print("setting up ssh")
 		winVM.Credentials.SetSSHKey(f.Signer)
 		if err := winVM.GetSSHClient(); err != nil {
 			return nil, fmt.Errorf("unable to get ssh client for vm %s : %v", instanceID, err)
 		}
-		if err := winVM.SetupWinRMClient(); err != nil {
-			return nil, fmt.Errorf("unable to setup winRM client for vm %s : %v", instanceID, err)
-		}
```

Comment:
Removing WinRM should not be a separate commit. This will cause reviewers to ask questions like I did for the previous commit where you are moving the WinRM code around which  makes reviewing more difficult.